home/models.py

```diff
@@ -24,18 +25,51 @@
 )
 
 
+@register_setting
+class SiteSettings(BaseSetting):
+    content_variations_options = StreamField(
+        [
+            ("content_variations_options", blocks.CharBlock()),
+        ],
+        blank=True,
+        null=True,
+    )
+
+
 class MediaBlock(AbstractMediaChooserBlock):
     def render_basic(self, value, context=None):
         pass
 
 
+def get_choices():
+    if SiteSettings.objects.first():
+        return [
+            (choice, choice)
+            for choice in SiteSettings.objects.first().content_variations_options
```

Comment:
We shouldn't have SSiteSettings.objects.first here, we should get the site settings for the correct site

Reply:
Follow-up PR has changed this to use the settings for the default site as discussed